docs/INSTALLATION.md

```diff
@@ -0,0 +1,89 @@
+
+![Speaker wiring](imgs/manual/zone_speaker_wiring.png)
+
+AmpliPro's amplifiers are capable of driving 4-8 Ohm speaker loads in a stereo configuration. Here is what a typical stereo speaker connection, using CL2 rated 14-AWG speaker wire and the Pheonix connectors, looks like:
+
+![Pheonix connector connection](imgs/manual/speaker_wire_to_pheonix.png)
+
+To connect a stereo speaker pair, using speaker wire:
+
+1. Strip 3-4 inches of the cable jacket, then strip 1/4 inch of insulation from the end of each wire.
+2. Twist the frayed end of each wire to keep things nice and neat.
+3. Unscrew each set screw to open the contacts, then insert and tighten down each wire one by one. Note that set screws loosen, but should not come out.
+4. To avoid any shorts, make sure that there aren't any stray wire strands.
+5. The speaker set can now be connected to one of the 6 zones.
+
+### Connections
+- Improper connection of speakers can damage the unit
+- Never connect mutiple speakers in parallel such that it brings the total impedance below 4 ohms (max 2x 8ohm speakers in parallel).
+- Amplified speaker ouputs **CAN NOT** be bridged, attempting to do so will damage the amplifier and void the warranty
+
+### Runnning wire in walls and ceilings
+- Most electric codes require the use of CL2 rated speaker wire for in-wall installations. Please refer to local building codes for more details.
+- Avoid running speaker wire next to AC power wires as much as possble to reduce noise.
+- When necessary, cross AC wires at 90 degree angles to avoid introducing any noise into the speakers.
+
+### More Info
+
+Much more information on speaker selection and installation can be the found in AmpliPro’s online documentation and question forums found at the end of this section.
+
+## Preamp Outputs
+Each of AmpliPro's zone outputs have a corresponding line-level preout output pair. These outputs can be used to connect powered subwoofers and other active/powered speakers.
+
+The volume output of a preout is controlled by the corresponding speaker output. This allows a connected subwoofer to be controlled propoertionally to the speakers output in the same zone.
+
+## Audio Inputs
+Each of the stereo RCA inputs can be connected to a different audio source, such as the output of a TV or a record player.
+Expansion Units
+To increase the number of zones you can add expansion units to your system. You can add up to 5 additional zone expander units to a single AmpliPro main unit. Each expansion unit adds 6 zones or pairs of speakers. Zones attach to main units using the CHAIN IN/OUT connectors on the rear panel.These are included with each Zone Expander.
+
+## Network connection
+Connect an RJ45 cable to the Ethernet port on the main unit. Connect the other end to a port on your main network (likeliy on your router). The unit's IP address is configured by DHCP by default.
```

Comment:
Can users set a static ip in the config?

Reply:
@linknum23